static-site/content/featured-analyses.yaml

```diff
@@ -4,16 +4,16 @@
 # - When adding or modifying descriptions, ensure it takes up at most two lines
 #   when rendered on the splash page.
 
+- name: HPAI outbreaks
+  description: "HPAI outbreak dynamics in North America: 2021-present"
+  img: hpai_in_na.png
+  url: /community/narratives/moncla-lab/nextstrain-narrative-hpai-north-america@main/HPAI-in-North-America
```

Comment:
Noting that this doesn't work because the narrative references a groups dataset, and those don't work on Heroku review apps due to insufficient AWS permissions.

This could be tested by manually deploying to dev.nextstrain.org or just merging and seeing if it works on next.nextstrain.org.

Reply:
I noticed that, but I verified the URL is correct by editing the heroku hostname to just be "nextstrain.org", and verifying that the dataset loads.

Reply:
> Noting that the URL doesn't work in preview because the narrative references a groups dataset

Oh, interesting that the narrative is via /community but the dataset is via /groups. I wonder if @lmoncla knows that narratives can be uploaded to groups too?

Reply:
@tsibley I did not know that! To be honest, we had some confusion trying to figure out how to get this narrative public, so very happy for any suggestions. Also going to tag @ajaeger207 here who put this together

Reply:
Nod. What you've done is just fine, but it's a bit easier with Groups. It's like uploading a dataset, e.g.

```shell
nextstrain remote upload \
    https://nextstrain.org/groups/moncla-lab/narratives/HPAI-in-North-America \
    hpai-north-america-narrative.md
```

There's a way author images in the Markdown too so that they're embedded in the narrative automatically upon upload instead of having to author an `<img>` reference an external URL on GitHub.

We should update the narratives documentation and https://github.com/nextstrain/nextstrain-narratives-demo (which I see was used).